models/articles.models.js

```diff
@@ -10,4 +10,27 @@ exports.selectArticleById = (article_id) => {
         return rows[0]
         }
     })
-}
\ No newline at end of file
+}
+
+exports.selectArticles = () => {
+    return db.query(`ALTER TABLE articles
+    ADD COLUMN comment_count INT
+    DEFAULT 0;
```

Comment:
Unfortunately we can't go back and just alter the original table here. Think about how we can first SELECT the articles table, then JOIN the comments table in order to get the comment_count that we want 👍